### ISSUE TITLE
Training with EfficientNet0

### ISSUE THREAD

**Issue by kwade4** [kwade4/FloraAI#6]
Initial training using:

- EfficientNet0
- Adam optimizer
- Learning rate of 1e-3
-  500 epochs

**Comment by kwade4** [comment]
The batch loss is oscillating, so maybe the learning rate is too high. Also looking into learning rate schedulers.

**Comment by kwade4** [comment]
With AdamW, learning rate scheduler, and pre-trained weights, validation accuracy is ~98%.

**Comment by kwade4** [comment]
Best performing model had initial learning rate of 5e-4 and gamma of 0.85.

**Comment by kwade4** [comment]
Excluding the images with watermarks gives a slightly lower validation accuracy (~98.1%). 

- Optimizer: AdamW
- Learning rate: 5e-4
- Learning rate scheduler: StepLR with step=2 and gamma=0.85
- Trained for 50 epochs
